src/mlpack/tests/README.md

```diff
@@ -43,4 +43,4 @@ To run a single test, you can explicitly provide the name of the test, for examp
 
 `./bin/mlpack_test BinaryClassificationMetricsTest`
 
-Catch2 provides many other features like filter, checkout the [Catch2 reference section](docs/Readme.md#top) - for more details.
+Catch2 provides many other features like filter, checkout the [Catch2 reference section](https://github.com/catchorg) - for more details.
```

Comment:
Should we link to here instead? https://github.com/catchorg/Catch2/blob/devel/docs/Readme.md#top

Reply:
yes that is more convenient .. i am changing : )